packages/react-experiments/src/components/UnifiedPicker/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './UnifiedPicker';
+export * from './UnifiedPicker.types';
+export * from './hooks/useFloatingSuggestionItems';

```

Comment:
Now that I'm looking at this, I'm thinking that it might be a better idea to put these hooks into the FloatingSuggestionsComposite directory, then import it here and in the editable item file. That way any consumers of just the picker could use that built in set of keyboard nav if they wanted.